DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/EditDelegateController.cs

```diff
@@ -72,6 +72,13 @@ public IActionResult Index(EditDelegateFormData formData, int delegateId)
                 );
             }
 
+            ProfessionalRegistrationNumberHelper.ValidateProfessionalRegistrationNumber(
+                ModelState,
+                formData.HasProfessionalRegistrationNumber,
+                formData.ProfessionalRegistrationNumber,
+                true

```

Comment:
`true` matches the same as the optional parameter. Do we want to keep it explicit here for readability?